stdlib/LinearAlgebra/test/runtests.jl

```diff
@@ -7,6 +7,4 @@ end
 
 @testset "Docstrings" begin
     undoc = Docs.undocumented_names(LinearAlgebra)
```

Comment:
```suggestion
    @test isempty(Docs.undocumented_names(LinearAlgebra))
```
You forgot the `@test`

Reply:
Thank you, @stevengj. I have incorporated this comment as follows:

@testset "Docstrings" begin 
     undoc = Docs.undocumented_names(LinearAlgebra) 
     @test undoc == [ ] 
 end 

I will work on the documentation early next week and submit it for your review.

Reply:
The style we've used elsewhere is `@test isempty(Docs.undocumented_names(LinearAlgebra))`.  Please follow the same style here.  

Also, there is no need to assign it to a variable that is used only once.  That is, please do it exactly the way I suggested.

In general, if you are willing to accept a suggestion as-is, the best way to resolve a conversation like this is to click the "commit suggestion" button to merge the commit (then pull it to your local branch if you are editing elsewhere).  That way, it is clear from the github interface that the suggestion has been incorporated, rather than forcing reviewers to manually check that you've made the change exactly as suggested.

Reply:
Thank you, @stevengj for reviewing this and for the advice on the process. I will follow this from next time on.

My motivation was that in case someone wants to make a quick fix in the future for an undocumented symbol, this could be done quickly by adding to the `[]`. However, I also note that the required change is very small and the suggested method is much cleaner.

Reply:
In #52723 it was decided to use `isempty` style.

Hopefully, in the future there will be no need to add new undocumented symbols to the list, because this test will fail on any PR that attempts to export an undocumented symbol.  That is the whole point of this test — undocumented symbols should be caught *before* they are merged in the future.

Reply:
Perfect! Thank you!